### ISSUE TITLE
collectstatic warning

### ISSUE THREAD

**Issue by flibbertigibbet** [azavea/cac-tripplanner#86]
When running `manage.py collectstatic`, get warning:

```
/opt/app/python/cac_tripplanner/destinations/forms.py:8: RemovedInDjango18Warning: 
Creating a ModelForm without either the 'fields' attribute or the 'exclude' attribute is 
deprecated - form DestinationForm needs updating
  class DestinationForm(ModelForm):
```

**Comment by CloudNiner** [comment]
Just added a fix for this in #84

**Comment by flibbertigibbet** [comment]
Spiffy.
